MIT.md

```diff
@@ -0,0 +1,7 @@
+## Copyright 2022, [Awesome Books]
```

Comment:
- [OPTIONAL] I think it would be better to write your name instead of the name of the project 👌